source/variables/transitions.css

```diff
@@ -0,0 +1,10 @@
+:root {
+	/* The amount of time for transitions to take. */
```

Comment:
https://github.com/connectivedx/fuzzy-chainsaw/issues/111
Please see my open question about mixin and what I had to say about the transition mixin. More than happy to kill this now, but left it just in case my above list was incorrect.